atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/workflow/SingleRowTwoCellsWorkflowConfiguration.java

```diff
@@ -27,7 +27,7 @@
 @JsonDeserialize(as = ImmutableSingleRowTwoCellsWorkflowConfiguration.class)
 @JsonTypeName(SingleRowTwoCellsWorkflowConfiguration.TYPE)
 public interface SingleRowTwoCellsWorkflowConfiguration extends WorkflowConfiguration {
-    String TYPE = "singleRowTwoCells";
+    String TYPE = "single-row-two-cells";
```

Comment:
nice

Reply:
yeah not sure how I missed it the first time around